fxsampler/src/main/java/fxsampler/FXSampler.java

```diff
@@ -58,18 +48,29 @@
 import javafx.scene.control.TreeView;
 import javafx.scene.layout.GridPane;
 import javafx.scene.layout.Priority;
+import javafx.scene.layout.StackPane;
 import javafx.scene.layout.VBox;
+import javafx.scene.web.WebEngine;
 import javafx.scene.web.WebView;
 import javafx.stage.Screen;
 import javafx.stage.Stage;
 import javafx.util.Callback;
-import fxsampler.model.EmptySample;
-import fxsampler.model.Project;
-import fxsampler.model.SampleTree.TreeNode;
-import fxsampler.model.WelcomePage;
-import fxsampler.util.SampleScanner;
 
-public class FXSampler extends Application {
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.ServiceLoader;
+
+public final class FXSampler extends Application {
+    
+    private static final String CACHE_RESULT = "CACHE_RESULT";
```

Comment:
Maybe add a little comment above explaining that it serves to indicate that a tab has already been loaded?

Reply:
I have renamed the field to `TAB_LOAD_CACHE`